src/java.base/share/classes/javax/crypto/CryptoPermissions.java

```diff
@@ -183,10 +183,10 @@ public boolean implies(Permission permission) {
     }
 
     /**
-     * Returns an enumeration of all the Permission objects in all the
-     * PermissionCollections in this CryptoPermissions object.
-     *
-     * @return an enumeration of all the Permissions.
+     * Returns an enumeration of all the {@code Permission} objects
+     * in all the PermissionCollections in this
```

Comment:
There is no PermissionCollections class. So, this should probably be changed to {@code PermissionCollection} objects.

Reply:
I don't think we need the middle part. Nothing is lost by removing it.